droidlet/lowlevel/hello_robot/hello_robot_mover.py

```diff
@@ -229,11 +229,11 @@ def move_relative(self, xyt_positions, blocking=True):
             xyt_positions = [xyt_positions]
         for xyt in xyt_positions:
             self.nav_result.wait()
-            self.nav_result = safe_call(self.nav.go_to_relative, xyt)
+            self.nav_result = safe_call(self.nav.go_to_relative, xyt, 10000000000, distance_threshold, angle_threshold)
             if blocking:
                 self.nav_result.wait()
 
-    def move_absolute(self, xzt_positions, blocking=True):
+    def move_absolute(self, xzt_positions, blocking=True, distance_threshold=None, angle_threshold=None):
         """Command to execute a move to an absolute position.
 
         It receives positions in canonical world coordinates and converts them to pyrobot's coordinates
```

Comment:
assuming None means that these are not used by default - but in that case how does it ever stop? does this have a timeout?

Reply:
None might not be a good default if so

Reply:
None will make it default to the planner's default values, which are dynamic based on the map resolution.
For example, the default planner's distance threshold is:
```
threshold = (float(self.map_resolution) - 1e-10) / 100.0
```

Reply:
this makes sense, because the planner cant plan a path with a resolution lower than it's map resolution anyways